### DISCUSSION TITLE
Getting a boolean field that determins if a record is related to another

### DISCUSSION THREAD

**Discussion by matteopolak** [prisma/prisma#17326] [Q&A] [answered]
I have a table that stores a bunch of records. For convenience, I'll call them comments.
I'm looking to add a "like" feature, and I have implemented it with a many-to-many relationship between `Comment`s and `User`s.
When retrieving comments by some metric (say, most recent), I would like to know if each comment was liked by a certain user.

From what I can see, I can only retrieve `Comment`s that are related to a `User`, or `User`s related to a `Comment`. Is there a way to retrieve every `Comment`, attaching some field or metadata that determines if it is related to a `User`? This would be used to show the "like" status of a comment for a given user.

Thanks,

**Comment by matteopolak** [comment, marked as answer]
Nevermind I've got it. I select all users by key and set the `liked` field if its length is 1 on the server-side before sending it to the client.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
